cdap-app-fabric/src/test/java/co/cask/cdap/AppWithMultipleScheduledWorkflows.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap;
+
+/**
+ *
+ */
+public class AppWithMultipleScheduledWorkflows extends AppWithMultipleWorkflows {
```

Comment:
I needed an app with multiple workflows (that were not scheduled). so this refactoring seemed like a good way to do it.
Otherwise, I can copy the existing app, and remove just the schedules from it.